django_project/ta_system/views.py

```diff
@@ -1,6 +1,14 @@
 from django.shortcuts import render, redirect
+from .models import SystemStatus
 from django.contrib.auth.decorators import login_required
 
+
 @login_required
 def home(request):
-    return render(request, 'ta_system/home.html')
+    date_list = SystemStatus.objects.order_by('id')
+    date_list = date_list.reverse()
+    status_dict = {
+        'system_status': date_list,
```

Comment:
Is the system_status actually being used in home.html? Seems like only the latest status is being used.

Reply:
code deleted